identity/realm-export.json

```diff
@@ -408,6 +408,29 @@
   "webAuthnPolicyPasswordlessAvoidSameAuthenticatorRegister": false,
   "webAuthnPolicyPasswordlessAcceptableAaguids": [],
   "users": [
+    {
+      "id" : "6a4ccf58-14a7-4c68-8f35-9107f98755b2",
+      "createdTimestamp" : 1678334999731,
+      "username" : "admin",
+      "enabled" : true,
+      "totp" : false,
+      "emailVerified" : false,
+      "firstName" : "admin ",
+      "lastName" : "admin",
+      "credentials" : [ {
+        "id" : "0bed1acd-4585-450d-9fca-0cd02fb57140",
+        "type" : "password",
```

Comment:
Hi @lnghia, what is the password of admin user?
Can you set password as password ? and 
"firstName" : "Join",
"lastName" : "Doe",